buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -82,7 +84,9 @@ SPECS = ['aix_ppc-64_cmprssptrs'      : CURRENT_RELEASES,
          'linux_x86-64_cmprssptrs_cmake' : ['9'],
          'win_x86'                    : ['8'],
          'win_x86-64_cmprssptrs'      : CURRENT_RELEASES,
-         'zos_390-64_cmprssptrs'      : ['11']]
+         'zos_390-64_cmprssptrs'      : ['11'],
+         'osx_x86-64'                 : ['8', '11'],
+         'osx_x86-64_cmprssptrs'      : ['8', '11']]
```

Comment:
This setting controls the releases and platforms for the 'nightly' builds.  We should likely not add 8 until we have machines capable of performing 8 builds.  No eta on when we get a machine(s) capable of 8 builds.